libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -229,11 +228,31 @@ export class DropdownComponent
     }
   }
 
+  /* Utility that makes it easier to set styles on card element 
+  when using popover*/
+  private setPopoverCardStyle(style: string, value: string) {
+    if (!this.usePopover) return;
+
+    this.renderer.setStyle(
+      this.cardElement.nativeElement,
+      style,
+      value,
+      RendererStyleFlags2.DashCase
+    );
+  }
+
   ngAfterViewInit() {
+    if (this.usePopover && this.expand === 'block') {
+      const { width } = this.elementRef.nativeElement.getBoundingClientRect();
+      this.setPopoverCardStyle('--kirby-card-width', `${width}px`);
+      this.setPopoverCardStyle('max-width', 'initial');
+      this.setPopoverCardStyle('min-width', 'initial');
```

Comment:
Are these necessary when we set them in the dropdown.component.scss?

Reply:
Yes! 😄
The selector `:host.expand kirby-card` only targets the card if `usePopover = false`. This is because we're moving `Kirby-popover` with `kirby-card` slotted to be a direct child of the `body` element, as far as I know.  

I'm open for suggestions on a better way to do it however; the current solution is not particularly pretty or very `css/html > javascript`-esque.